DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/ToHost/Raw/PolycurveToHostSplineRawConverter.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.AutocadShared.ToHost.Raw;
+
+/// <summary>
+/// Polycurve segments might appear in different ICurve types which requires to handle separately for each segment.
+/// </summary>
+public class PolycurveToHostSplineRawConverter : IRawConversion<SOG.Polycurve, List<ADB.Entity>>
+{
+  private readonly IRawConversion<SOG.Line, ADB.Line> _lineConverter;
+  private readonly IRawConversion<SOG.Polyline, ADB.Polyline3d> _polylineConverter;
+  private readonly IRawConversion<SOG.Arc, ADB.Arc> _arcConverter;
+  private readonly IRawConversion<SOG.Curve, ADB.Curve> _curveConverter;
+
+  public PolycurveToHostSplineRawConverter(
+    IRawConversion<SOG.Line, ADB.Line> lineConverter,
+    IRawConversion<SOG.Polyline, ADB.Polyline3d> polylineConverter,
+    IRawConversion<SOG.Arc, ADB.Arc> arcConverter,
+    IRawConversion<SOG.Curve, ADB.Curve> curveConverter
+  )
+  {
+    _lineConverter = lineConverter;
+    _polylineConverter = polylineConverter;
+    _arcConverter = arcConverter;
+    _curveConverter = curveConverter;
+  }
+
+  public List<ADB.Entity> RawConvert(SOG.Polycurve target)
+  {
+    // POC: Should we join entities?
```

Comment:
I think we can improve this when we can have a IIndex of raw converters probably and then your case statement can go, Add a POC: to that effect so we can use the IIndex, I don't know about this other question, that probably needs to stay

Reply:
added poc comment